src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -178,68 +184,79 @@
     <section id="document-delivery-section" class="mt-10">
       <header>
         <h2>{{getHeaderNumber('documentDelivery')}}. Dissolution Documents Delivery</h2>
-        <p class="mt-1">Copies of the dissolution documents will be sent to the following email addresses listed
-          below:
+        <p class="mt-4 mb-6">
+          Copies of the dissolution documents will be sent to the following email addresses listed below:
         </p>
       </header>
-      <div :class="{ 'invalid-section': isDocumentDeliveryInvalid }">
-        <DocumentDelivery
-          :editableCompletingParty="isRoleStaff"
-          :showCustodianEmail="true"
-          :invalidSection="isDocumentDeliveryInvalid"
-          :registeredOfficeEmail="getBusinessContact.email"
-          :custodianEmail="getCustodianEmail"
-          :userEmail="getUserEmail"
-          :documentOptionalEmail="getDocumentDelivery.documentOptionalEmail"
-          @update:optionalEmail="setDocumentOptionalEmail($event)"
-          @valid="setDocumentOptionalEmailValidity($event)"
-        />
-      </div>
+      <DocumentDelivery
+        :class="{ 'invalid-section': isDocumentDeliveryInvalid }"
```

Comment:
Unfortunately, some sub-components need this here, and some have it on a div inside the component.